src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -325,6 +326,11 @@ network::network(program::ptr program, stream::ptr stream, bool is_internal, boo
         wait_for_the_turn();
     }
 
+    _kernels_cache = std::unique_ptr<kernels_cache>(new kernels_cache(program->get_engine(), program->get_id(),
```

Comment:
I think we only need to have this when the network is dynamic?

Reply:
Done